index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="keyword" content="Apple Main Web Page" />
+    <meta name="author" content="Uchenna Anya and Henoks Mossissa" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+    <title>Apple Main Web Page</title>
+  </head>
+  <body>
+    <header class="custom-header">
+      <nav class="nav">
+        <a href="" class="nav-elem apple">
+          <span
+            class="iconify"
+            data-icon="cib:apple"
+            data-inline="false"
+          ></span>
+        </a>
+        <a href="#" class="nav-elem"> Store </a>
+        <a href="#" class="nav-elem"> Mac </a>
+        <a href="#" class="nav-elem"> iPod </a>
+        <a href="#" class="nav-elem"> iPhone </a>
+        <a href="#" class="nav-elem"> iPad </a>
+        <a href="#" class="nav-elem"> iTunes </a>
+        <a href="#" class="nav-elem"> Support </a>
+      </nav>
+      <div class="nav-search">
+        <form>
+          <div class="search">
+            <span
+              class="iconify"
+              data-icon="ic-sharp-search"
+              data-inline="false"
+            ></span>
+            <input
+              type="search"
+              name="search"
+              class="search-space"
+              id="search"
+            />
+          </div>
+        </form>
+      </div>
+    </header>
+    <main>
+      <section class="front">
+        <div class="text">
+          <h1>What will your verse be?</h1>
+          <h2>
+            Everything has something to share. See how people are using iPad in
+            their unique ways to add to the world's story.
+          </h2>
+          <div class="watch">
+            <a href="#" class="clickable">
+              Watch the "your verse" film and explore the stories.
+            </a>
+            <span
+              class="iconify"
+              data-icon="codicon:play-circle"
+              data-inline="false"
+            ></span>
+          </div>
+        </div>
+      </section>
+      <div class="promo">
+        <figure>
+          <img
+            src="assets/images/promo_iphone5s.jpg"
+            alt="iphone5s"
+            class="promo-img"
+          />
+        </figure>
+        <figure>
+          <img
+            src="assets/images/promo_iphone5c.jpg"
+            alt="iphone5c"
+            class="promo-img"
+          />
+        </figure>
+        <figure>
+          <img
+            src="assets/images/promo_30_years.jpg"
+            alt="30years"
+            class="promo-img"
+          />
+        </figure>
+        <figure>
+          <img
+            src="assets/images/promo_supplier_responsibility.jpg"
+            class="promo-img"
+            alt="responsibility"
+          />
+        </figure>
```

Comment:
Each of these items (images) should be linked. Kindly wrap it around the `<a>` tag. 
![21:01:37](https://user-images.githubusercontent.com/26856491/94609640-fbee9900-0296-11eb-88d8-2051ac003218.png)